File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
	int UniqueElements = 1;
       	if (nums.empty())
		return 0;
	for (int i = 1; i < nums.size(); i++)
	{
		if (nums[i] != nums[i - 1])
		{
			nums[UniqueElements] = nums[i];
			UniqueElements++;
		}
	}
	return UniqueElements; 
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.ts
function removeDuplicates(nums: number[]): number {
    let j=1;
    let i=1;
    while(j<nums.length){
        if(nums[j]!=nums[j-1]){
            nums[i]=nums[j];
            i++;
        }
        j++;
    }
    return i;
};